specs/lib/MergeDuplicatePropertiesSpec.js

```diff
@@ -139,8 +139,6 @@ describe('MergeDuplicateProperties', function() {
 
     var mergeShaders = MergeDuplicateProperties.mergeShaders;
     describe('mergeShaders', function() {
-        var testShaderBufferOne = Buffer.from('test shader one', 'utf8');
-        var testShaderBufferTwo = Buffer.from('test shader two', 'utf8');
         it('merges duplicate shaders', function() {
```

Comment:
This test is basically useless, why doesn't it should load and use real gltf files, not mock what it expects the source to look like.

Given you want to get out a new release, I'm okay with @likangning93 merging this, but I'll write up an issue to fix this and other tests that do this.